docs/tutorials/create-react-app.mdx

```diff
@@ -0,0 +1,325 @@
+
+The first thing we need to do, is go into our nitric.yaml file and add the new site.
+
+```yaml
+name: nitric-greeter
+services:
+  greeter:
+    path: greeter
+    runtime: function/nodets12
+
+sites:
+  greeter-ui:
+    path: greeter-ui
+    assetPath: ./build
+    buildScripts:
+      - yarn build
+```
+
+This new addition to the Nitric stack tells the `nitric run` command to look for a site at the path `/greeter-ui`, run the build script `yarn build` and then look for the assets in the `./build` directory. When `yarn build` is run, it will run `react-scripts build` which builds our react app into a ./build directory. `yarn` can be replaced with `npm` if that is your preferred package manager.
+
+We can then define entrypoints to our app
+
+```yaml
+name: nitric-greeter
+services:
+  greeter:
+    path: greeter
+    runtime: function/nodets12
+
+sites:
+  greeter-ui:
+    path: greeter-ui
+    assetPath: ./build
+    buildScripts:
+      - yarn build
+
+entrypoints:
+  main:
+    paths:
+      /:
+        type: site
+        target: greeter-ui
+      /greeter/:
+        type: service
+        target: greeter  
+```
+
+This tells the Nitric CLI how to build the paths and where they point. Our first path will be the site, which will be at the root. The next path is the /greeter/ path, which is where the greeter service is hosted. This greeter service will sit relative to the site and so we can use our function that way.
+
+This can be done using a fetch statement in a new function that is called by our button. The fetch statement will make a HTTP request to our service and we will receive back our response.
+
+```typescript
+// /src/App.tsx
+import React, { useState } from 'react';
+
+const App = () => {
+  const [name, setName] = useState("Nitric");
+  const [greeting, setGreeting] = useState("Hello")
+  const [response, setResponse] = useState("");
+
+  async function getResponse() {
+    const rawResponse = await fetch("/greeter/",{
+      method: 'POST',
+      body: JSON.stringify({
+        name: name,
+        greeting: greeting
+      })
+    });
+    const content = await rawResponse.json();
+    setResponse(content.response);
+  }
+  return (
+    <div>
+      <form>
+        <label>
+          Name:
+          <input type="text" value={name} onChange={e => setName(e.target.value)}></input>
+        </label>
+        <br/>
+        <label>
+          Greeting:
+          <input type="text" value={greeting} onChange={e => setGreeting(e.target.value)}></input>
+        </label>
+      </form>
+      <button onClick={getResponse}>Greetings World!</button>
+      <h1>{response}</h1>
+    </div>
+  );
+}
+
+export default App;
+
+```
+
+If we now use our `nitric run` command we should get a URL at the bottom. If we go to this URL in our browser it will open up into our app.
```

Comment:
This tutorial is awesome :). My only suggestion would be to add a screen capture of the final result.